김현수/mvvmstudy/app/src/main/java/com/example/mvvmstudy/IO/RetrofitClass.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.mvvmstudy.IO
+
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import java.util.concurrent.TimeUnit
+
+object RetrofitClass {
+    var retrofitService : IGithubService
```

Comment:
retrofitService을 singleton으로 사용하려고 한다면 `var` 이 아닌
val로 Lazy 선언해도 될 것 같아요!  그런데 개인적인 생각으로는 retrofitService 객체를 여기서 참조하지 않고 추후 Hilt등 DI에서 Singleton으로 구현하면 될 것 같아요! 참고만 해주세요!
```suggestion
    val retrofitService : IGithubService by lazy { }
```